Sources/StringStyle.swift

```diff
@@ -173,7 +174,21 @@ extension StringStyle {
                 return attributedString
```

Comment:
#? @KingOfBrian this is working, but how? If we return here, we won't ever get to the transform. I built this the other night, and everything seems to be working, but I can't see how.